### ISSUE TITLE
Drag and Drop Timeline for external events drop without mouseposition

### ISSUE THREAD

**Issue by Clarence344** [almende/vis#2327]
Hey,

I have lot of external items that I want to drag and drop into the group timeline.
I want to drop an item with fix time, start and end. 

like:

 ```
var item = {
      id: new Date(),
      type: itemType,
      content: event.target.innerHTML.trim(),
      start: new Date(2016,11,17,12,0,0,0),
      end: new Date(2016,11,17,22,0,0,0)
    };
```

But if I drop the item into the timeline they always takes the position of the mouse pointer for the start time. 

How can I solve this problem?

**Comment by yotamberk** [comment]
So the idea was when i've first implemented it was to purposly not set an item by its start time, but solely by the mouse position. I do think that if a time is supplied, it should set it accordingly. But still not implemented yet. 
 I think this feature is needed. I'll take a look at implementing it the next few days when i have the time :-)

**Comment by Clarence344** [comment]
Thanks a lot, I'm looking forward to hear from you soon
But why includes the item options "start" and "end", I dont understand that :)

**Comment by yotamberk** [comment]
The initial thought was to simply use the difference between the times to determine the length of the item. 
I didn't put too much thought in to it obviously, hoping the opensource community will help me better understand this new feature I was introducing.
Basically I was feeling out drag and drop. I'm happy people are interested in it.

**Comment by yotamberk** [comment]
I'm also looking at D&D for groups. I want to see if there is interest in that too

**Comment by Clarence344** [comment]
thanks. I need this important feature to bring my project to the end :)

**Comment by Clarence344** [comment]
Hi @yotamberk have you already a solution for my problem? thanks

**Comment by yotamberk** [comment]
Not yet... I'm working on other critical bugs. If you can pick this up, that would be great. 
Hopefully I can do this for next week

**Comment by yotamberk** [comment]
Closed with #2372
